rbac/server/api/main.py

```diff
@@ -125,17 +111,12 @@ def main():
         resources={r"/api/*": {"origins": "*"}, r"/webhooks/*": {"origins": "*"}},
     )
 
-    zmq = ZMQEventLoop()
-    asyncio.set_event_loop(zmq)
-    server = app.create_server(
```

Comment:
See [this](https://sanic.readthedocs.io/en/latest/sanic/api_reference.html?highlight=create_server).

> This does not support multiprocessing and is not the preferred way to run a Sanic application.